scripting.md

```diff
@@ -0,0 +1,104 @@
+# SuperTux Level Editor Scripting Tutorial
+
+## What is Scripting?
+
+Scripting in SuperTux is when a game object is altered using things like a script trigger or a switch.  Scripting is written in the Squirrel coding language.
+
+#### What sort of things use scripting?
+
+Scripts can be activated by the following things, plus many more:
+
+* Script Trigger, which activates the script.
+* Switch/Button, which activate scripts when used.
+* Death Scripts, which activate when an enemy dies.
+* Init-Scripts, which activate when Tux enters the sector.
+
+In order to edit a script, right click a scriptable object and open the script editor.
+
+## Structure of scripts
+
+The basic structure of a line of script is as so:
+
+    object.action(Number);
+
+"object" is what you are going to activate, "action" is what the object does, and "Number" is the property of the action, like a number, series of numbers, or words. The semicolon activates the script, you have to use it in most situations.
```

Comment:
Use <code>\`object\`</code> rather than double quotes, so that it displays as `object`. Same for most other things here (action, Number, elsewhere).